index.html

```diff
@@ -0,0 +1,138 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mental Health Awareness Global Event</title>
+  <link rel="stylesheet" href="home.css"/>
+  <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.13.0/css/all.css" />
+  <link href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;1,100;1,300;1,400&display=swap" rel="stylesheet">
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet"/>
+  <script src="home.js" defer></script>
+</head>
+
+<body>
+  <div class="admin-div">
+    <ul class="admin-list">
+      <li><a href="#" class="admin-item"><img src="./Images/facebook-app-symbol.png" class="facebook" alt="facebook"></a></li>
+      <li><a href="#" class="admin-item"><img src="./Images/twitter.png" class="twitter" alt="twitter"></a></li>
+      <li><a href="#" class="admin-item">English</a></li>
+      <li><a href="#" class="admin-item">My Page</a></li>
+      <li><a href="#" class="admin-item">Logout</a></li>
+  </ul>
+  </div>
+  <div class="nav-div">
+    <nav>
+      <img src="./Images/Icon-Cancel.png" alt="Cancel" class="cancel-img" >
+      <a href="http://127.0.0.1:5500/index.html" class="logo-nav"><img src="./Images/Logo.png" class="logo-nav" alt="logo"></a>
+      <ul class="intro-list">
+        <li><a href="http://127.0.0.1:5500/about.html" class="navi-item">About</a></li>
```

Comment:
- [ ] The link is broken in the online version of your website. I would recommend recommend removing `http://127.0.0.1:5500` because it's link of the local server, and replacing it by `.`.
```html
  <li><a href="./about.html" class="navi-item">About</a></li>
```